crates/fluvio-connector-package/src/lib.rs

```diff
@@ -1,3 +1,15 @@
+use secret::resolve_str_secrets;
+use serde_yaml::Value;
+
 pub mod metadata;
 pub mod config;
 pub mod secret;
+
+pub fn resolve_config(input: Value) -> anyhow::Result<Value> {
+    let str_input = serde_yaml::to_string(&input)?;
+    let str_result = resolve_str_secrets(&str_input)?;
+    // Here we can resolve envs, files, etc.
+
+    let result = serde_yaml::from_str(&str_result)?;
+    Ok(result)
+}
```

Comment:
@galibey I think that we can add to this function all the resolvers that we need. For now I only added secret resolver